hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -379,6 +379,9 @@
             sleep the peripheral can continue its signaling even in deep sleep. The testbench
             verifies the correctness of the reflected values once the chip goes into deep sleep.
             This is replicated for DIO pins as well.
+
+            In this test, passthrough feature is not tested. The feature is
+            covered in other tests such as chip_sw_sleep_pwm_pulses.
             '''
       stage: V2
       tests: []
```

Comment:
ah just a nit, you might want to update the tests here now also.

Reply:
ah i see..it's not complete yet, that's why you didn't populate it. sounds good.

Reply:
yep. I will add the test when it is ready